fapolicy_analyzer/ui/actions.py

```diff
@@ -22,6 +22,7 @@
 
 INIT_SYSTEM = "INIT_SYSTEM"
 SYSTEM_INITIALIZED = "SYSTEM_INITIALIZED"
+ERROR_SYSTEM_INITIALIZATION = "ERROR_SYSTEM_INITIALIZATION"
```

Comment:
After discussing with @tparchambault I'm ok with leaving this as is since this constant already exists in the `strings` module. This will avoid confusion between the two.

Reply:
Name clash with existing strings module variable. Off-line conversation w/ @dorschs57 to leave it for readability/maintainability.